packages/frontend/src/components/organisms/forms.scss

```diff
@@ -1,3 +1,34 @@
+
 .responsiveTableContainer{
   overflow-x:auto;
 }
+.form {
+  .input {
+    margin: .5rem 0 .5rem 0;
+    border-radius: 12px;
+    &:focus {
+      outline: none;
+      border-color:#ff581a; 
+      border-width: 2px;
+      box-shadow: none;
+    }
+  }
+  &.login {
+    width: 50vw;
+    margin:auto;
+    background-color: #fff;
+    padding: 2rem;
+    -webkit-box-shadow: 4px 4px 8px 0px rgba(0,0,0,0.1);
+    -moz-box-shadow: 4px 4px 8px 0px rgba(0,0,0,0.1);
+    box-shadow: 4px 4px 8px 0px rgba(0,0,0,0.1);
+  }
+}
+
+// Small devices (landscape phones, less than 768px)
+@media screen and (max-width: 768px){
+  .form {
+    &.login {
+      width: 100%;
+    }  
+  }
+}
```

Comment:
The line jump is still missing ;)

Reply:
Before you said " there **mustn't** be a line jump. Re read what you write before criticizing. <3